integration-cli/docker_api_set_unix_test.go

```diff
@@ -0,0 +1,43 @@
+// +build !windows
```

Comment:
@jhowardmsft are we testing with windows clients against linux servers?  I thought we were.